cylc/flow/scripts/lint.py

```diff
@@ -326,7 +326,7 @@ def check_cylc_file(file_, checks, modify=False):
                     print(
                         Fore.YELLOW +
                         f'[{message["purpose"]}{message["index"]:03d}]'
-                        f'{file_}: {line_no}: {message["short"]}'
+                        f' {file_rel}:{line_no}: {message["short"]}'
```

Comment:
Use relative filepaths in reporting (ala flake8) to make the summary easier to read.